shared/js/background/tab-manager.es6.js

```diff
@@ -198,6 +199,8 @@ class TabManager {
         const tab = tabManager.get({ tabId: request.tabId })
 
         if (tab) {
+            tab.status = request.status
+            tab.site.status = tab.status
```

Comment:
I think this tab.site.status can go in the bin now we have tabState to track it all :)